media/css/mozorg/home/home-pocket-best-2022-promo.scss

```diff
@@ -0,0 +1,106 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-pocket';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.pocket-best-2022-promo {
+    background-color: #dceaff;
+    padding: 0;
+
+    .pocket-best-2022-promo-title {
+        @include text-display-sm;
```

Comment:
**non-blocking, question**: I don't think I've seen the `display` mixins in use, are they meant to be deprecated? https://github.com/mozilla/protocol/blob/7cbfb52acef30c67f324202db187c09992f071b1/assets/sass/protocol/includes/mixins/_typography.scss#L151

Reply:
Ah, so they are  - will update to `title`!